### DISCUSSION TITLE
Cannot save entities for tire pressure

### DISCUSSION THREAD

**Discussion by Spacerat76** [ngocjohn/vehicle-status-card#26] [Q&A] [answered]
If i select an entity for the front left & right tire and switch to rear, then the front entries are gone.
Same if I press save.

How can i configure the tire card?
![grafik](https://github.com/user-attachments/assets/d94efe4a-3010-4381-bc76-2f444a9a3e90)

**Comment by ngocjohn** [comment]
Hi @Spacerat76 , I have tested it on myself now, it seems to be without problems. The data is displayed even if only 2 front wheels are entered. Do you have the latest version of the card? Or can you test in a new card? 
![CleanShot 2024-10-28 at 22 33 46](https://github.com/user-attachments/assets/60a33612-5656-433e-b8c7-400a3ba3aad9)
![CleanShot 2024-10-28 at 22 33 59](https://github.com/user-attachments/assets/d3d51f28-d0d1-4f71-954d-44f73f9285a1)

**Reply by Spacerat76** [reply]
I had v1.3.0

**Comment by Spacerat76** [comment, marked as answer]
V1.3.1 fixed the issue
